default/stringtables/en.txt

```diff
@@ -4277,14 +4277,18 @@ Robotic species are sapient machines. These species' max population can be incre
 PHOTOTROPHIC_SPECIES_TITLE
 Phototrophic Metabolism
 PHOTOTROPHIC_SPECIES_TEXT
-'''Phototrophic species live mainly or entirely on sunlight. These species' max population is greatly influenced by the brightness of the local star - the bigger the better. Growth specials do not benefit them.
+'''Phototrophic species live mainly or entirely on sunlight. These species' max population is greatly influenced by the brightness of the local star - the bigger the better - and the planet size. Growth specials do not benefit them.
```

Comment:
The "brightness of the local star and the planet size - the brighter or bigger, the better." sentence won the contest. :)